packages/ses/src/module-load.js

```diff
@@ -348,7 +348,7 @@ function asyncJobQueue() {
    *
    * @returns {Promise<Array<Error>>}
    */
-  const drainQueue = async () => {
+  async function drainQueue () {
```

Comment:
this could conditionally be an async arrow function on engines that support it (i.e. not Hermes), if really needed